src/NzbDrone.Automation.Test/Sonarr.Automation.Test.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFrameworks>net6.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Selenium.Support" Version="4.8.1" />
+    <PackageReference Include="Selenium.Support" Version="4.8.2" />
```

Comment:
This causes conflict issues with 4.0+. Reverted to 3.X in dev

Reply:
@Qstick, fixed